examples/org.eclipse.glsp.example.workflow/src/org/eclipse/glsp/example/workflow/layout/WorkflowLayoutEngine.java

```diff
@@ -31,8 +34,17 @@
    public void layout() {
       if (modelState.getRoot() instanceof GGraph) {
          GLSPLayoutConfigurator configurator = new GLSPLayoutConfigurator();
-         configurator.configureByType("graph");
-         this.layout((GGraph) modelState.getRoot(), configurator);
+         configurator.configureByType("graph")
+            .setProperty(LayeredOptions.NODE_PLACEMENT_BK_FIXED_ALIGNMENT,
```

Comment:
I would also set the `EDGE_ROUTING` property to `polyline`. Otherwise we create a lot of unnecessary bendpoints that we have to remove afterwards.
```suggestion
            .setProperty(LayeredOptions.EDGE_ROUTING, EdgeRouting.POLYLINE),
            .setProperty(LayeredOptions.NODE_PLACEMENT_BK_FIXED_ALIGNMENT,
```